e2e/dractions/crud.go

```diff
@@ -207,3 +223,15 @@ func generateDRPCDiscoveredApps(name, namespace, clusterName, drPolicyName, plac
 
 	return drpc
 }
+
+func annotateDRPCDoNotDeletePVC(ctx types.TestContext, managementNamespace, drpcName string) error {
```

Comment:
Remove the management namespace since the drpc and vrg are not always in the same namespaces.